### PR TITLE
set-git-credentials 1.0.0

### PR DESCRIPTION
**Pull request by dphillipsprosper**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [X] __I will not move an already shared step version's tag to another commit__
- [X] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [X] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [X] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [X] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

This is my first custom bitrise step. I know it's incredibly simple, but it was a good exercise in learning how to create custom steps, and I also think it adds value to the community. The current documentation, [Setting your Git credentials on build machines](https://devcenter.bitrise.io/builds/setting-your-git-credentials-on-build-machines/), instructs you to manually create a Script step and copy a script to do it into it. With this, now it's as simple as adding another workflow step and editing a couple textboxes.

### CODE REVIEW

**Comment by dphillipsprosper** [comment]
Reading the Step Development Guideline, not sure when I should add the icon for the step. Should I do it now in this PR, or should it be a separate PR?

File: steps/set-git-credentials/1.0.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+- ubuntu-16.04
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
+  - name: wget
```
Comment (koral--) [inline steps/set-git-credentials/1.0.0/step.yml]: Why wget is here? It does not seem to be used.

**Comment by koral--** [comment]
> Reading the Step Development Guideline, not sure when I should add the icon for the step. Should I do it now in this PR, or should it be a separate PR?

It is OK to add icon file in the same PR.

File: steps/set-git-credentials/1.0.0/step.yml (hunk 1 above)
Comment (dphillipsprosper) [reply steps/set-git-credentials/1.0.0/step.yml]: Good question - I assume it was added by default when I generated the step template using `bitrise :step create`. I can remove it.

**Comment by dphillipsprosper** [comment]
@koral-- I've removed wget, and added the icon we'd like to go with.

**Comment by BirmacherAkos** [comment]
@godrei could you please merge the PR?

**Pull request merged**